ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -18,6 +18,8 @@
 #include "ibtk/FECache.h"
 #include "ibtk/FEDataManager.h"
 #include "ibtk/IBTK_CHKERRQ.h"
+#include "ibtk/IndexUtilities.h"
+#include "ibtk/JacobianCalculator.h"
```

Comment:
I need to change these over to `<` and `>` at some point...

Reply:
I always thought that the "rule" was to use angle brackets for system headers, quotes for non-system headers.

Reply:
The only difference is that using `"`s also includes the current directory in the preprocessor path. Ultimately its not that big of a deal but I prefer to use the slightly stricter pathing rule.